gulpfile.js

```diff
@@ -180,11 +181,18 @@ gulp.task('docs:mkdocs', shell.task([
   'mkdocs serve'
 ], {
   cwd: 'docs'
-}))
+}));
+
+gulp.task('lint', function () {
+  return gulp.src(['src/pathfora.js', 'gulpfile.js', 'test/pathforaSpec.js', 'docs/docs/examples/**/*.js'])
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
 
 gulp.task('test', ['build:styles', 'build:testjs']);
 gulp.task('build:local', ['build:styles', 'local:js']);
```

Comment:
probably worth adding the lint task to any of the build function since it should run against the compiled files not the source files. any dynamic aspects, like {{templates}} will not pass linter and wrapping in `''` will break the js once its rendered.